.markdownlint-cli2.yaml

```diff
@@ -0,0 +1,10 @@
+# Reference: https://github.com/DavidAnson/markdownlint-cli2#markdownlint-cli2yaml
+
+config:
+  ul-indent:
+    # Kramdown wanted us to have 3 earlier, tho this CLI recommends 2 or 4
```

Comment:
Just noting here that I think 2 is the most common and would thus be least surprising. As we currently use 3 I am still fine continuing with that at least through the transition. After that we could maybe think about switching to 2, this is what I would prefer.

Reply:
Yes I agree. We have 95% of the lines as 3 now (counting all repos) and probably 80% even in this repo, since it was that way with the old linter. 

We can definitely do 2 later on, to get rid of this last custom thing we have. After that, we basically have no global rules or config, so we're doing the defaults only, which is optimal.